LoopKitUI/Extensions/Environment+Authenticate.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Environment+Authenticate.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 8/3/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LocalAuthentication
+import SwiftUI
+
+public typealias AuthenticationChallenge = (_ description: String, _ completion: @escaping (Result<Void, Error>) -> Void) -> Void
+public extension Result where Success == Void {
```

Comment:
Ah, just noticed this extension is public. I'm not sure it makes sense for LoopKitUI to vend this as public API.